src/mlpack/methods/ann/layer/dropout.hpp

```diff
@@ -85,22 +81,26 @@ class Dropout
                 arma::Mat<eT>&& g);
 
   //! Get the input parameter.
-  InputDataType const& InputParameter() const { return inputParameter; }
+  InputDataType const& InputParameter() const { return inputParameter;  }
+
   //! Modify the input parameter.
   InputDataType& InputParameter() { return inputParameter; }
 
   //! Get the output parameter.
   OutputDataType const& OutputParameter() const { return outputParameter; }
+
```

Comment:
usually we group those two functions together.

Reply:
You mean the call by value and call by reference ones @zoq ?

Reply:
You mean the call by value and call by reference ones @zoq ?

Reply:
yes